subsys/bluetooth/controller/Kconfig

```diff
@@ -206,6 +206,21 @@ config BT_CTLR_ISO_TX_BUFFER_SIZE
 	  Size of the Isochronous Tx buffers and the value returned in HCI LE
 	  Read Buffer Size V2 command response.
 
+config BT_CTLR_ISOAL_LOG_LEVEL
+	int "ISO-AL logging level"
+	range 0 4
+	default 3
+	help
+	  Use this option to enable ISO-AL logging at the given level.
+	  Default is LOG_LEVEL_INF (3).
```

Comment:
Instead of doing this, you should use the template for modules as per https://docs.zephyrproject.org/latest/services/logging/index.html#logging-in-a-module

So it would be something like 

```c
module = BT_CTLR_ISOAL
module-str = "ISO-AL"
source "subsys/logging/Kconfig.template.log_config"
```

Reply:
Should be done now. Not exactly the same because it looked as if there was need to support legacy logging as well.